operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 ## 4.1.0
 
 * [CHANGE] Configured `max_total_query_length: 12000h` limit to match Mimir jsonnet-based deployment. #3879
+* [FEATURE] Add cache support for GEM's admin bucket. #3740
```

Comment:
Can you change the place of this changelog entry please? This is in the 4.1.0 section which was already released

Reply:
also nit: if you have a link to the GEM docs for this feature and/or if you can add why a user might enable this feature. That would make it slightly easier for users to know if this feature concerns them.